### ISSUE TITLE
Can I upload an image with text?

### ISSUE THREAD

**Issue by costales** [piroor/tweet.sh#18]
Hi,

Is possible to upload an image and some text together?

Thanks in advance!

**Comment by roygbyte** [comment]
Here is some code I am using for a project. Hopefully it helps!

```
# This script assumes you have tweet.sh in the same directory
# ... as well as a folder named "images" with a JPEG named "image.jpg"

# Upload image and assign output to a variable
MEDIA_JSON=$(./tweet.sh upload images/image.jpg)

# Create a file to store the results from the request, then store it there.
touch request.json
echo "$MEDIA_JSON" > request.json

# Get the media ID from the request
# Basically, use a regular express to gather sequences of numbers that are 19 digits long
# Then grab the first sequence that is returned using `head -n 1`
MEDIA_ID=$(cat $DIR/request.json | grep -o '[0-9]\{19\}' | head -n 1)

# Create our tweet and link it with the media id
./tweet.sh post -m $MEDIA_ID "Your string here"
```

**Comment by piroor** [comment]
With 42e91c1 now you can specify images with `-i` option(s) like:

```
$ ./tweet.sh post -i ~/photos/photo1.jpg -i ~/photos/photo2.png my photo.
```
